src/_pytest/python.py

```diff
@@ -1153,21 +1157,20 @@ def _find_parametrized_scope(argnames, arg2fixturedefs, indirect):
     return "function"
 
 
-def _idval(val, argname, idx, idfn, config=None):
+def _idval(val, argname, idx, idfn, item, config=None):
```

Comment:
a required item makes the optional config unneeded for now it seems

Reply:
True, fixed.